bin.ts

```diff
@@ -2,12 +2,25 @@
 
 import { QuickImportCycleChecker } from './src/quick-import-cycle-checker';
 
-// TODO Make this more elegant/configurable
-const directoryToCheck = process.cwd();
-const rootDirectory = process.cwd();
+export interface cliSettings {
+    checkDirectories: string[];
+    root: string;
+    exclusions: RegExp[];
+}
 
-QuickImportCycleChecker.forDirectories(directoryToCheck)
-    .withRootDirectory(rootDirectory)
+const settings: cliSettings = {
+    checkDirectories: process.env.npm_config_checkdirectories?.split(',') ?? [process.cwd()],
```

Comment:
... Oof, wasn't aware that this is possible! :O Learned something new, thanks!
I guess my comment here is off then? https://github.com/abuob/quick-import-cycle-checker/pull/17/files#r921471403

In any case... does this still work when using the pass-through hyphens?
If not, I think just parsing the `process.argv.slice(2)` is a bit the more traditional approach